packages/plugins/apollo-managed-federation/src/index.ts

```diff
@@ -27,64 +25,91 @@ export type ManagedFederationPluginOptions = (
 };
 
 export function useManagedFederation(options: ManagedFederationPluginOptions = {}): Plugin {
-  const {
-    supergraphManager = new SupergraphSchemaManager(options as SupergraphSchemaManagerOptions),
-  } = options;
+  let _supergraphManager: SupergraphSchemaManager;
+
+  function ensureSupergraphManager() {
+    if (_supergraphManager == null) {
+      if (options instanceof SupergraphSchemaManager) {
+        _supergraphManager = options;
+      } else {
+        _supergraphManager = new SupergraphSchemaManager(options);
+      }
+    }
+    return _supergraphManager;
+  }
```

Comment:
This looks like a breaking change isn't it ?
Before, the manager was provided through `options.supergraphManager`, now it's the `options` object itself that should be the manager.

Reply:
Yes it is a breaking change, and the plugin package has a `minor` changeset because it is v0.